pygmt/src/select.py

```diff
@@ -128,6 +131,38 @@ def select(data=None, outfile=None, **kwargs):
         and **+i** reverses the tests to pass record with *z* value NOT in the
         given range. Finally, if **+c** is not used then it is automatically
         incremented for each new ``z_subregion`` argument, starting with 2.
+    dist2pt : str
+        *pointfile*\|\ *lon*/*lat*\ **+d**\ *dist*.
+        Pass all records whose locations are within *dist* of any of the
+        points in the ASCII file *pointfile*. If *dist* is zero, the 3rd
+        column of *pointfile* must have each point's individual radius of
+        influence. If you only have a single point you can specify
+        *lon*/*lat* instead of *pointfile*. Distances are Cartesian and in
+        user units. Alternatively, if ``region`` and ``projection`` are used,
```

Comment:
I am unsure about the **-fg** option in the [upstream GMT documenation](https://docs.generic-mapping-tools.org/dev/gmtselect.html#c):
> Distances are Cartesian and in user units; specify -fg to indicate spherical distances and append a distance unit, even if the distance specified is 0.